### ISSUE TITLE
NNBD bug: type 'Null' is not a subtype of type 'String' in type cast

### ISSUE THREAD

**Issue by hoc081098** [lejard-h/google_maps_webservice#99]
```
[ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: type 'Null' is not a subtype of type 'String' in type cast
E/flutter (16725): #0      _$PlaceDetailsFromJson (package:google_maps_webservice/src/places.g.dart:136:20)
E/flutter (16725): #1      new PlaceDetails.fromJson (package:google_maps_webservice/src/places.dart:710:7)
E/flutter (16725): #2      _$PlacesDetailsResponseFromJson (package:google_maps_webservice/src/places.g.dart:294:26)
E/flutter (16725): #3      new PlacesDetailsResponse.fromJson (package:google_maps_webservice/src/places.dart:838:7)
E/flutter (16725): #4      GoogleMapsPlaces._decodeDetailsResponse (package:google_maps_webservice/src/places.dart:528:29)
E/flutter (16725): #5      GoogleMapsPlaces.getDetailsByPlaceId (package:google_maps_webservice/src/places.dart:126:12)
E/flutter (16725): <asynchronous suspension>
E/flutter (16725): #6      displayPrediction (package:example/main.dart:136:18)
E/flutter (16725): <asynchronous suspension>
E/flutter (16725): #7      _MyAppState._handlePressButton (package:example/main.dart:120:5)
E/flutter (16725): <asynchronous suspension>
```

**Comment by hoc081098** [comment]
Same #93

**Comment by hoc081098** [comment]
Fixed  in `^0.0.20-nullsafety.2` 😃
